project/ProjectPlugin.scala

```diff
@@ -20,28 +21,48 @@ object ProjectPlugin extends AutoPlugin {
 
     lazy val pluginSettings = Seq(
       sbtPlugin := true,
+      crossSbtVersions := Seq(sbtV.`0.13`, sbtV.`1.0`),
       resolvers ++= Seq(
         Resolver.sonatypeRepo("snapshots"),
         "jgit-repo" at "http://download.eclipse.org/jgit/maven"),
-      addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.0"),
-      addSbtPlugin("com.typesafe.sbt" % "sbt-site"    % "1.2.0"),
-      addSbtPlugin("org.tpolecat"     % "tut-plugin"  % "0.5.2"),
+      addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.2"),
+      addSbtPlugin("com.typesafe.sbt" % "sbt-site"    % "1.3.0"),
       libraryDependencies ++= Seq(
+        "com.47deg"             %% "org-policies-core" % "0.6.1-SNAPSHOT",
```

Comment:
This is `0.6.1` and `sbt-dependencies` is `0.6.1`. This is because the plan is to release in this order?
* sbt-org-policies: 0.6.0
* sbt-dependencies using 0.6.0
* sbt-org-policies: 0.6.1
* sbt-microsites using 0.6.1

Reply:
I was originally using a 0.6.1-SNAPSHOT locally to test these changes.
I was deciding releasing sbt-org-policies: 0.6.0 with a snapshot version of sbt-dependencies or releasing versions with no snapshot dependencies.

I like your way better. There is probably going to be at least one more version bump for all three sbt plugins after sbt-microsites gets published.